hbase-zookeeper/pom.xml

```diff
@@ -139,6 +139,15 @@
       <artifactId>log4j-slf4j-impl</artifactId>
       <scope>test</scope>
     </dependency>
+    <!-- ZooKeeperServer needs the below dependencies, thus MiniZooKeeperCluster also needs them -->
+    <dependency>
+      <groupId>org.xerial.snappy</groupId>
+      <artifactId>snappy-java</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>commons-cli</groupId>
+      <artifactId>commons-cli</artifactId>
```

Comment:
I think this one can be at test scope.

Reply:
We expose MiniZooKeeperCluster as IA.Public and it is under src/main, not src/test, so we must declare the dependencies which are only used by zookeeper server at compile/runtime scope...

IIRC there is a issue for removing the class but there are objections from our users...